src/config.ts

```diff
@@ -323,15 +323,26 @@ exports.krsProviders = {
   }
 };
 
-exports.bitcoinAverageBaseUrl = 'https://apiv2.bitcoinaverage.com/indices/local/ticker/';
+const bitcoinAverageBaseUrl = 'https://apiv2.bitcoinaverage.com/indices/local/ticker/';
 
 // TODO: once server starts returning eth address keychains, remove bitgoEthAddress
-exports.defaultConstants = (env) => {
+const defaultConstants = (env) => {
 
   if (common.Environments[env] === undefined) {
     throw Error(`invalid environment ${env}`);
   }
 
   const network = common.Environments[env].network;
-  return _.merge({}, defaults, exports.tokens[network]);
+  return _.merge({}, defaults, tokens[network]);
+};
+
+export = {
+  tx,
+  chains,
```

Comment:
same thing here, can be simplified by adding `export` in front of variables/functions declarations